.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "external/imgui"]
 	path = samples/imgui_display/imgui
 	url = https://github.com/ocornut/imgui.git
-[submodule "external/websocketpp"]
```

Comment:
why did you remove this?

Reply:
Oh I accidently remove this while removing ptcl. I will rollback this.